### PR TITLE
Add onnx metadata to classification models

### PR DESCRIPTION
**Pull request by sovrasov**
### Summary
This PR adds metadata to output onnx classification models. This metadata can be used by model_api to infer onnx model directly without conversion to IR.
Corresponding PR to mapi: https://github.com/openvinotoolkit/model_api/pull/106

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Review by sungmanc — DISMISSED** [review]

File: src/otx/algorithms/common/utils/utils.py [hunk 1]
```diff
@@ -153,3 +154,16 @@ def read_py_config(filename: str) -> adict:
     )
 
     return cfg_dict
+
+
+def embed_onnx_model_data(onnx_file: str, extra_model_data: Dict[Tuple[str], Any]) -> None:
```
Comment (eunwoosh) [inline src/otx/algorithms/common/utils/utils.py:159]: Could you add unit test for this?

**Review by eunwoosh — COMMENTED** [review]
Thanks for your work. I found that corresponding PR is still draft. Then should this PR be merged after corresponding PR is merged?

**Comment by sovrasov** [comment]
> Thanks for your work. I found that corresponding PR is still draft. Then should this PR be merged after corresponding PR is merged?

This PR doesn't depend on MAPI PR.

File: src/otx/algorithms/common/utils/utils.py (hunk 1 above)
Comment (sovrasov) [reply src/otx/algorithms/common/utils/utils.py:159]: Done

**Review by goodsong81 — APPROVED** [review]

**Review by jaegukhyun — APPROVED** [review]

**Review by eunwoosh — APPROVED** [review]

**Pull request merged**